lib/kube/proxy/portforward_test.go

```diff
@@ -556,7 +557,14 @@ func TestPortForwardUnderlyingProtocol(t *testing.T) {
 			readyCh := make(chan struct{})
 			// errCh receives a single error from ForwardPorts goroutine.
 			errCh := make(chan error)
-			t.Cleanup(func() { require.NoError(t, <-errCh) })
+			t.Cleanup(func() {
+				err := <-errCh
```

Comment:
This may block the whole test suite. Are you sure it won't be an issue and errCh will always be readable?